src/main/java/net/spring/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package net.spring.service;
+
+import net.spring.entity.user.User;
+
+public interface UserService {
+    User findByUserName(String userName);
```

Comment:
better name findByUserName -> findByName

you already have word "User" in service name